bazel/external/http-parser.BUILD

```diff
@@ -4,4 +4,5 @@ cc_library(
     hdrs = ["http_parser.h"],
     includes = ["."],
     visibility = ["//visibility:public"],
+    copts = ["-DHTTP_MAX_HEADER_SIZE=61440"],
```

Comment:
To avoid surprises, at least for now, can we change this back to what it was before? I think it was 80K or so? Can you check? This will end up being a risky change otherwise (even though I think it should be fine). I might go as far as to say that we should split this one out into its own PR as it's much higher risk?

Reply:
I'll move the http-parser change into another PR. I'm struggling to get the MarkupSafe change to pass tests as the layout of the tar has changed. Can't work out why local  tests pass. Will close this and break into separate PR's. Thanks.

Reply:
@mattklein123 `61440` is what's enforced in Envoy: https://github.com/envoyproxy/envoy/blob/87553968ec2258919d986e9a76512b0009d01575/source/common/http/conn_manager_impl.cc#L618
you're right, `byteSize()` might be calculated differently than `HTTP_MAX_HEADER_SIZE`, so leaving it at `81920` might be indeed a good idea. Thanks!

Reply:
@mattklein123 the default is 80K - https://github.com/nodejs/http-parser/blob/master/http_parser.h#L63. Will check the Envoy default of 60K that @PiotrSikora mentioned and do some research.

Reply:
@moderation oops, sorry! I thought they lowered it to 8kB in [nodejs/http-parser](https://github.com/nodejs/http-parser) and that's why the tests were failing, but it seems that they only did that in [nodejs/node](https://github.com/nodejs/node)...

Please ignore the `copts` and forget I said anything.